pyfar/classes/audio.py

```diff
@@ -706,9 +706,9 @@ def __init__(
 
         # check fft norm
         if fft_norm in self._VALID_FFT_NORMS:
-            if self._complex and fft_norm == "rms":
-                raise ValueError(("'rms' normalization is not valid for "
-                                  "complex time signals"))
+            if self._complex and fft_norm in ["rms", "power"]:
```

Comment:
how about ``'psd'``?

Reply:
This is part of #567 - should be fixe after that one is merged...